mlflow/store/model_registry/databricks_workspace_model_registry_rest_store.py

```diff
@@ -29,6 +29,19 @@ def get_model_version_by_alias(self, name, alias):
             "and try again.",
         )
 
+    def copy_model_version(self, src_mv, dst_name):
+        """
+        Copy a model version from one registered model to another as a new model version.
+
+        :param src_mv: A :py:class:`mlflow.entities.model_registry.ModelVersion` object representing
+                       the source model version.
+        :param dst_name: the name of the registered model to copy the model version to. If a
+                         registered model with this name does not exist, it will be created.
+        :return: Single :py:class:`mlflow.entities.model_registry.ModelVersion` object representing
+                 the cloned model version.
+        """
+        return self._copy_model_version_impl(src_mv, dst_name)
```

Comment:
For the workspace model registry, I have a PR in progress that enables creating model versions using model version URIs.